tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -267,7 +268,7 @@ IncludeByKey.PASSTHROUGH {
 				}
 				Object.Widget.gain.1 {
 					Object.Control.mixer.1 {
-						name 'Main Capture Volume 1'
+						name 'Post Mux $SSP0_PCM_NAME Capture Volume'
```

Comment:
@jsarha you're hitting the 44 char limit with this one right? Maybe it will be fixed when you removed the widget name from the name?

Reply:
Yes. I decided to ignore those cases that that that would be Ok without the widget name prefix. However, I am open for other suggestions for naming standard that would fit into smaller space, still be informative enough, and would fulfill this: https://docs.kernel.org/sound/designs/control-names.html